README.md

```diff
@@ -33,8 +29,7 @@ The samples were developed and tested using [Mars 4.5.0](https://eclipse.org/mar
 
 ## Using remote input PDF files with the samples
 
-By default, the samples run with pre-packaged input PDF files that are treated as resources of the samples. However, you can use your own input PDF files 
-including those that are stored remotely on a shared network with the samples. As long as the shared network where the PDF files reside is mounted on your local machine it's treated a part of your local file system.
+By default, the samples run with pre-packaged input PDF files that are treated as resources of the samples. However, you can use your own input PDF files, including those that are stored remotely on a shared network with the samples. As long as the shared network where the PDF files reside is mounted on your local machine it's treated a part of your local file system.
```

Comment:
I did not change the content. When I opened the Readme.MD file in MarkDown, the formatting looked awkward, so I removed the line break to use word wrap.